eth/stagedsync/stage_mining_create_block.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/ledgerwatch/log/v3"
 
 	"github.com/ledgerwatch/erigon-lib/kv"
-	"github.com/ledgerwatch/erigon-lib/txpool"
+	"github.com/ledgerwatch/erigon/zk/txpool"
```

Comment:
I see your point though, this is around "traditional" block production rather than zk specifically.  I have a feeling this was needed as the same type is passed around in a few places, will take a look into it

Reply:
Had another look just now.  Because we've changed the txpool type on the `Ethereum` type in `backend.go` to support the couple of changes for ZK these files have been affected as they take in this struct directly.  I don't think it would be clean to import two txpool packages and have 2 txpools and 2 txpool DBs if you agree?